std_msgs/Header header
uint64 id
vslam/MapPoint[] map_points
vslam/KeyFrame[] key_frames
geometry_msgs/Pose camera_pose
